apps/web/src/services/administration/iam/policy/policy-detail/PolicyDetailPage.vue

```diff
@@ -65,12 +66,14 @@
                 </div>
             </div>
             <div class="policy-detail-contents">
-                <p-field-title>{{ $t('IAM.POLICY.FORM.ID') }}</p-field-title>
+                <p-field-title class="policy-type">
```

Comment:
nitpick: The class name 'policy-type' doesn't seem to fit well here.😅 How about using 'policy-detail-field-title' or else?